source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts

```diff
@@ -37,11 +37,20 @@ export class BerlinContainer extends HostContainer {
 		header.className = "berlin-header";
 		outerFrame.appendChild(header);
 
-		const frame = document.createElement("div");
-		frame.className = "berlin-inner-container";
-		frame.appendChild(this.cardHost);
+		const headerText = document.createElement("p");
+		headerText.innerHTML = "Berlin Header";
```

Comment:
`innerHTML` should avoided in *almost* all circumstances. in this case, `textContent` is the more appropriate property to manipulate. it's not consequential in this instance, but it's a best practice...

if you're curious as to why, consider what happens if the string here someday gets wired up to untrusted input -- someone can inject arbitrary HTML into our payload (think `<script>`).

```suggestion
		headerText.textContent = "Berlin Header";
```